From 5a066fa63bbb02a6280111dc5c5d2e44dabe061a Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Thu, 18 Oct 2018 09:38:38 +0200
Subject: [PATCH 1/2] conf/ucm/Dell-WD15-Dock: Fix incorrect device names

The device name string for Dell WD15 (and its variants) dock is set as
"WD15Dock", while the actual device name to be used is "Dock".

Bugzilla: https://bugzilla.suse.com/show_bug.cgi?id=1112292
Fixes: 8ebb40c96970 ("conf/ucm: Add a UCM profile for Dell WD15 Dock USB-audio")
Signed-off-by: Takashi Iwai <tiwai@suse.de>
(cherry picked from commit 6eced6b4b6dca9f62ea3d13a40c5991231df3c5c)
Signed-off-by: Hui Wang <hui.wang@canonical.com>
---
 src/conf/ucm/Dell-WD15-Dock/HiFi.conf | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/conf/ucm/Dell-WD15-Dock/HiFi.conf b/src/conf/ucm/Dell-WD15-Dock/HiFi.conf
index b07f3440..1b6bd212 100644
--- a/src/conf/ucm/Dell-WD15-Dock/HiFi.conf
+++ b/src/conf/ucm/Dell-WD15-Dock/HiFi.conf
@@ -3,7 +3,7 @@ SectionDevice."Headphone" {
 
 	Value {
 		PlaybackChannels "2"
-		PlaybackPCM "hw:WD15Dock,0"
+		PlaybackPCM "hw:Dock,0"
 	}
 }
 
@@ -12,7 +12,7 @@ SectionDevice."LineOut" {
 
 	Value {
 		PlaybackChannels "2"
-		PlaybackPCM "hw:WD15Dock,1"
+		PlaybackPCM "hw:Dock,1"
 	}
 }
 
@@ -21,6 +21,6 @@ SectionDevice."Mic" {
 
 	Value {
 		CaptureChannels "2"
-		CapturePCM "hw:WD15Dock,0"
+		CapturePCM "hw:Dock,0"
 	}
 }
-- 
2.19.1

